recipes-multimedia/alsa/imx-alsa-plugins_git.bb

```diff
@@ -20,7 +20,7 @@ inherit autotools pkgconfig use-imx-headers
 PV = "1.0.26+${SRCPV}"
 
 SRC_URI = "git://github.com/nxp-imx/imx-alsa-plugins.git;protocol=https;branch=${SRCBRANCH}"
-SRCBRANCH = "MM_04.09.01_2408_L6.6.y"
+SRCBRANCH = "MM_04.09.02_2410_L6.6.y"
 SRCREV = "b2ba082e70333f187972ee4e85f63f9d2f608331"
```

Comment:
the branch changes but the SRCREV does not change?

Reply:
@zelan-nxp Please see the @angolini comments.